### ISSUE TITLE
ERR_UNSUPPORTED_DIR_IMPORT 

### ISSUE THREAD

**Issue by drewhoffer** [pacocoursey/cmdk#326]
Came here from using ShadCns Command component and randomly started seeing this after having no issues day prior. Im running a simple next app with:

```
    "cmdk": "^1.0.3",
    "next": "14.2.15",
    "react": "^18",
    "react-dom": "^18",
```

Blows up with the following:

```
 ⨯ [TypeError: [ERR_UNSUPPORTED_DIR_IMPORT] Directory import 'file:///frontend/node_modules/use-sync-external-store/shim' is not supported resolving ES modules imported from 'file:///frontend/node_modules/cmdk/dist/index.mjs'] {
  code: "ERR_MODULE_NOT_FOUND",
  page: "/"
}
```
Can provide any additional information that is needed.

**Comment by dotnize** [comment]
downgrading to fixed version `"1.0.2"` worked for me

**Comment by AdiRishi** [comment]
Same, ran into this error on version `1.0.3` with vite

```
Directory import '/Users/arishi/personal/asy-monorepo/node_modules/.pnpm/cmdk@1.0.3_@types+react-dom@18.3.1_@types+react@18.3.12_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/use-sync-external-store/shim' is not supported resolving ES modules imported from /Users/arishi/personal/asy-monorepo/node_modules/.pnpm/cmdk@1.0.3_@types+react-dom@18.3.1_@types+react@18.3.12_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/cmdk/dist/index.mjs
Did you mean to import "use-sync-external-store/shim/index.js"?
```

**Comment by drewhoffer** [comment]
Specifying 

> downgrading to fixed version `"1.0.2"` worked for me

Specifying 1.0.2 fixed for me as well.

**Comment by kachkaev** [comment]
Same here for Next.js 15.0.1 and cmdk 1.0.3 I went to `node_modules/cmdk/dist` and replaced `use-sync-external-store/shim` with `use-sync-external-store/shim/index.js`, as suggested in the error. It worked, so #328.

**Comment by pacocoursey** [comment]
Available in `1.0.4`

**Comment by dmagunov** [comment]
I still see an issue under version `1.0.4` in Vite
```
/node_modules/.pnpm/use-sync-external-store@1.2.2_react@18.3.1/node_modules/use-sync-external-store/shim/index.js?v=acd2932d' does not provide an export named 'useSyncExternalStore'
```

**Comment by lsmurray** [comment]
@pacocoursey - this led to an unintentional change in the types for 1.0.4 which would be a breaking change. I've provided a PR fix in #329 .
